packages/grid/src/vaadin-grid-data-provider-mixin.js

```diff
@@ -483,6 +483,10 @@ export const DataProviderMixin = (superClass) =>
      * @param indexes {...number} Row indexes to scroll to
      */
     scrollToIndex(...indexes) {
+      if (!this.__virtualizer || !this.clientHeight || !this._columnTree) {
```

Comment:
I suppose the check for ` !this.clientHeight || !this._columnTree` can be now removed from line below.

Reply:
Done.